src/App.css

```diff
@@ -0,0 +1,3 @@
+* {
+    font-family: 'Mark Pro';

```

Comment:
When using custom fonts, remember that they only work for you unless you've also added them to the whole app. In this case it looks like you've installed this, as I don't have it installed I'm seeing a completely different font.